docker-compose.yml

```diff
@@ -33,9 +45,22 @@ services:
       - "3000:3000"
       - "3001:3001"
     depends_on:
```

Comment:
should we add this for the worker_builder too?

Reply:
Those ports are only exposed to reach the API, they aren't needed for the worker.

Reply:
Ah gotcha 👍